Tasks/Common/PowershellHelpers/Invoke-ActionWithRetries.ps1

```diff
@@ -0,0 +1,93 @@
+function Invoke-ActionWithRetries {
```

Comment:
Please trim the trailing whitespace from this file. Example lines with trailing whitespace: 39, 45, 49, 55, and 57